tiles/src/main/java/com/protomaps/basemap/layers/Landuse.java

```diff
@@ -38,7 +38,8 @@ public static void processFeature(SourceFeature sf, FeatureCollector features, S
         .setAttr("place", sf.getString("place"))
         .setAttr("railway", sf.getString("railway"))
         .setAttr("sport", sf.getString("sport"))
-        .setZoomRange(5, 15);
+        .setZoomRange(5, 15)
+        .setMinPixelSize(3.0);
```

Comment:
There were a lot of tiny parks showing up too early causing the map to be too busy at zooms 10, 11, and 12. Adjusting this here and in Natural to a Minimum Pixel Size of `3.0` solves that. If you want slightly more noise then adjust to `2.0` instead.

Before:

<img width="671" alt="image" src="https://github.com/protomaps/basemaps/assets/853051/00be2cfa-a6d7-494f-87f0-3c2b3f97c953">

After:

<img width="745" alt="image" src="https://github.com/protomaps/basemaps/assets/853051/fe0f00d9-fe74-4212-91b1-f8caf1f5527f">

Reply:
LGTM!